app/assets/stylesheets/components.scss

```diff
@@ -272,13 +272,13 @@
 //checkpoints
 .primary-checkpoint {
   background-color: $shadow-grey;
-  bottom: 0;
+  border-radius: 4px;
```

Comment:
border-radius should not be changed, let it be 0!

Reply:
![Screenshot_20230218_231107](https://user-images.githubusercontent.com/96580571/219954894-14328e28-60d1-4154-a523-d609a2d06d03.png)

@vedant-jain03  done! 👍